network-api/networkapi/settings.py

```diff
@@ -462,30 +462,32 @@
 # LOGGING
 LOGGING = {
     'version': 1,
-    'disable_existing_loggers': False,
+    'disable_existing_loggers': True,
```

Comment:
This ensures we only use these loggers, so that production logging can be shut down (and handled by the Heroku router)

Reply:
I was checking Heroku recommended configuration for django project, and they keep the flag at [False](https://github.com/heroku/django-heroku/blob/master/django_heroku/core.py#L119). Are you sure we won't be missing any logs by silently silencing them?